suite-common/test-utils/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@suite-common/test-utils",
```

Comment:
Please add README to this package, about how to use this package.

Reply:
https://github.com/trezor/trezor-suite/pull/5815/commits/1783c364b7bba52fb55ac38fb98fa191a9fdc17b